From 36637599fd65c767b50bc6838d7da85af4279c07 Mon Sep 17 00:00:00 2001
From: yuehu mi <yuehu.mi@amlogic.com>
Date: Thu, 2 Jul 2020 14:41:02 +0800
Subject: [PATCH] pms: optimize pms service execution time [1/1]

PD#OTT-11440

Problem:
pms service scan apk takes time

Solution:
remove webview apk and android.software.webview.xml

Verify:
s905y2

Change-Id: Id72a7491c261b52485840ca58632f0aded142cb6
Signed-off-by: yuehu mi <yuehu.mi@amlogic.com>
---
 target/product/core_minimal.mk | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/target/product/core_minimal.mk b/target/product/core_minimal.mk
index b16f9f13c..598361d51 100644
--- a/target/product/core_minimal.mk
+++ b/target/product/core_minimal.mk
@@ -82,7 +82,6 @@ PRODUCT_PACKAGES += \
     uncrypt \
     vndk_snapshot_package \
     voip-common \
-    webview \
     webview_zygote \
 
 # Wifi modules
@@ -90,8 +89,6 @@ PRODUCT_PACKAGES += \
     wifi-service \
     wificond \
 
-PRODUCT_COPY_FILES += \
-    frameworks/native/data/etc/android.software.webview.xml:system/etc/permissions/android.software.webview.xml
 
 ifneq (REL,$(PLATFORM_VERSION_CODENAME))
 PRODUCT_COPY_FILES += \
-- 
2.25.0

